people/_posts/0000-04-01-mohamed-yousif.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Mohamed Yousif
+degree:
+position: Undergraduate student
+affiliation: Medical Biophysics
+cosupervisor:
+image:
+alumni: false
+
+# social
+email: myousif9@uwo.ca
+github: https://github.com/myousif9
+orcid:
+scholar: https://scholar.google.com/citations?user=i_xSNBkAAAAJ&hl=en&oi=ao
```

Comment:
For scholar, only need the part after `user=`, I think in this case `i_xSNBkAAAAJ`